torch/functional.py

```diff
@@ -331,9 +331,33 @@ def meshgrid(*tensors: Union[Tensor, List[Tensor]]) -> Tuple[Tensor, ...]:
         return _meshgrid(*tensors)
 else:
     def meshgrid(*tensors):
-        r"""Take :math:`N` tensors, each of which can be either scalar or 1-dimensional
-        vector, and create :math:`N` N-dimensional grids, where the :math:`i` :sup:`th` grid is defined by
-        expanding the :math:`i` :sup:`th` input over dimensions defined by other inputs.
+        r"""Creates grids of coordinates specified by the input tensors.
```

Comment:
How are the grids represented?

Reply:
I'm struggling to add more detail in the one-line summary. Do we have that goal for a succinct one-line summary that gives the gist?

I agree this doesn't really give the gist though.

Reply:
Don't kill yourself over it :P 

"Given N 1D tensors N_0 ... N_{N-1} as inputs with corresponding sizes S_0 ... S_{N-1}, this creates N N-dimensional tensors G_0 ... G_{N-1}, each with shape (S_0, ..., S_{N-1}) where the output G_i is constructed by expanding N_i to the to result shape."

Is that accurate? 

Then maybe a sentence that begins like "Conceptually..." or "This is helpful when..." that explains the mathematical perspective behind why this operation exists?

Reply:
I liked your text: it's quite direct and clear.

I added some text about where this is useful and an example that uses this to make a plot. PTAL!